src/angular/menu/menu.scss

```diff
@@ -44,6 +44,7 @@ sbb-menu {
   border-radius: var(--sbb-border-radius);
   box-shadow: var(--sbb-box-shadow-below);
   border-top-width: var(--sbb-border-width);
+  border-color: var(--sbb-color-weak-accent);
```

Comment:
Could you try modifying the panel border gloablly in `_typography.scss` instead of here? The select still has the wrong panel color.